demo/src/main/res/drawable/thumb.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item>
+    <shape android:shape="oval">
+      <solid android:color="@android:color/transparent" />
+      <size
+        android:width="24dp" android:height="24dp" />
+    </shape>
+  </item>
+
+  <item
+    android:top="6dp" android:right="6dp" android:left="6dp" android:bottom="6dp">
```

Comment:
`cmd-alt-l`? Should be one per line

Reply:
1f646cb4ad6185e55e1ac1327cfd37b86c4cafef